tests/dft-vv10/input.dat

```diff
@@ -27,3 +30,52 @@ scf_e, scf_wfn = energy("VV10", return_wfn=True)
 
 for k, v in bench.items():                        # TEST
     compare_values(v, psi4.get_variable(k), 9, k) # TEST
+scf_nl=psi4.get_variable('TOTAL SCF ENERGY')
+
+# dft-nl tests:
+
+set reference rks
+scf_e, scf_wfn = energy("BLYP-NL", return_wfn=True)
+compare_values(0.032852665973, psi4.get_variable('DFT VV10 ENERGY') , 6, 'BLYP-NL dashtest') # TEST
```

Comment:
Could you perhaps give the reference values sensible names and move them all to the top? Such as:
```
ref_blyp_nl_b_4 = 0.032852665973
ref_blyp_nl_b_5 = 0.0238582087067217
ref_blyp_nl_b_5_c_0_1 = 0.0246750
```

Reply:
ok